x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/token/CreateTokenRequest.java

```diff
@@ -34,6 +35,7 @@
 
     public enum GrantType {
         PASSWORD("password"),
+        KERBEROS("kerberos"),
```

Comment:
I wonder if we should call this `_kerberos` or something like that since it's not a standard OAuth2 grant type?
I'm not particular worried, but thought it might make sense.